### DISCUSSION TITLE
Option to hide "Effect" button on light more-info

### DISCUSSION THREAD

**Discussion by conorlap** [Mariusthvdb/custom-more-info#31] [Ideas]
**Is your feature request related to a problem? Please describe.**
I never use the Effect button, would be nice to remove it from the UI :) 

**Describe the solution you'd like**
Option to hide Effect button from light more-info

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by Mariusthvdb** [comment]
yes I can see the usefulness of that.

if we can find the Element to hide that would help developing the solution to the FR.

this would be it:

<img width="1074" alt="Scherm­afbeelding 2024-04-05 om 16 47 50" src="https://github.com/Mariusthvdb/custom-more-info/assets/33354141/5a5bbb44-2ea3-4741-8f32-ecc24bf8f47e">

changing:

<img width="611" alt="Scherm­afbeelding 2024-04-05 om 16 48 44" src="https://github.com/Mariusthvdb/custom-more-info/assets/33354141/084adf16-2f0c-40a5-a240-36a2777f2239">


to:

<img width="622" alt="Scherm­afbeelding 2024-04-05 om 16 48 29" src="https://github.com/Mariusthvdb/custom-more-info/assets/33354141/3c0d92a6-7173-4bd0-b8ab-ac6503d2abcc">

**Comment by Mariusthvdb** [comment]
btw, if you are looking for generic solutions, and not per entity (like this plugin provides) please have a look at the great Kiosk-mode plugin which was just updated to do what you want, see its latest release, and relevant PR https://github.com/NemesisRE/kiosk-mode/pull/221

**Comment by conorlap** [comment]
Thanks @Mariusthvdb  - the Kiosk-mode suggestion is working great 🥳

**Comment by Mariusthvdb** [comment]
Great!

if you dont mind, let's  keep this FR open, as I would love to see the per entity option.
